hack/scripts/releaser/releaser.go

```diff
@@ -73,10 +65,40 @@ type Release struct {
 	logger    logger.Logger
 	cmdRunner cmdrunner.CmdRunner
 
-	githubWorkflowID string
-	helmChartConfig  helmChart
+	helmChartConfig helmChart
 }
 
+func (r *Release) SaveReleaseInfo() error {
```

Comment:
Move below together with all other methods.
Methods are written below the object's constructor